### DISCUSSION TITLE
Ready for production use?

### DISCUSSION THREAD

**Discussion by marco910** [AxeWP/wp-graphql-rank-math#32] [General]
Hi there,

I just came across this plugin for WPGraphQL and want to use it on a production site. In the README it says that the plugin isn't ready for production use yet. Is this still the status, or is it recommended using this plugin in production already?

**Comment by justlevine** [comment]
Hey @marco910 

I'm currently aware of ~ a dozen sites currently using this plugin in production, without any problems. However, I'd personally wait until we release `v0.1.0` before recommending it, since we're still experimenting heavily with the ideal schema shapes and type definitions.

**Reply by marco910** [reply]
Hey @justlevine 

Thanks for letting me know!

**Comment by mattshoaf** [comment]
Hello @justlevine ! Seeing as the last update to this discussion is 2 years old, I'm curious if you have a timeline on the stable release yet? Thanks!

**Reply by justlevine** [reply]
Hey @mattshoaf , we released [v0.1.0 in March 2024](https://github.com/AxeWP/wp-graphql-rank-math/releases/tag/0.1.0), which means all our [releases](https://github.com/AxeWP/wp-graphql-rank-math/releases/) since are SemVer compliant, which should provide your frontend the schema stability necessary to use this in production. 

There's a few more additional modules that I want to expose, and probably some RM pro compatibility stuff before I bump this to a v1.0.0, but the stuff currently exposed is all production-tested.
